kin-sdk-core/src/main/java/kin/sdk/core/ClientWrapper.java

```diff
@@ -57,7 +57,7 @@ private KeyStore initKeyStore(Context context) {
         return new KeyStoreImpl(context);
     }
 
-    void wipeoutAccount() {
+    void wipeoutAccounts() {
```

Comment:
Do we need it?
please please check before a merge.

Reply:
will be considered lately in a different task, implementation added with integration tests task